packages.ini

```diff
@@ -331,6 +332,7 @@ validate_incorrect_missing_deps = six
 [djangorestframework-stubs==3.14.0]
 [djangorestframework-stubs==3.14.2]
 [djangorestframework-stubs==3.14.4]
+[djangorestframework-stubs==3.14.5]
```

Comment:
we use a fork of this in sentry -- we'll want to rebase that

Reply:
actually I've already rebased on this one!

Reply:
thank you